File: solutions/zig/two-fer/1/two_fer.zig
const std = @import("std");

pub fn twoFer(buffer: []u8, name: ?[]const u8) anyerror![]u8 {
    const person = name orelse "you";
    return std.fmt.bufPrint(buffer, "One for {s}, one for me.", .{person});
}

File: solutions/clojure/two-fer/1/src/two_fer.clj
(ns two-fer
  (:require [clojure.string :as str]))

(defn two-fer
  ([] (two-fer "you"))
  
  ([name]
   (let [target (if (str/blank? name) "you" name)]
     (str "One for " target ", one for me."))))
